schemas/openshift/terraform-output-format-1.yml

```diff
@@ -0,0 +1,43 @@
+---
+"$schema": /metaschema-1.json
+version: '1.0'
+type: object
+
+additionalProperties: false
+properties:
+  "$schema":
+    type: string
+    enum:
+    - /openshift/terraform-output-format-1.yml
+  provider:
+    type: string
+    enum:
+    - generic-secret
+  data:
+    type: object
+    additionalProperties: false
+    patternProperties: {
+      "^[a-zA-Z0-9_]{0,253}$": {
+        "type": "string"
+      }
+    }
+oneOf:
```

Comment:
Will this be expanded in the future with the well-known formats mentioned in https://gitlab.cee.redhat.com/service/app-interface/-/blob/master/docs/app-sre/design-docs/terraform-output-secret-format.md#well-known-formats?

Reply:
yes, well known formats could be dedicated providers or also something like
```yaml
terraformResources:
  - provider: aws-iam-service-account
    output_format:
      provider: well-known-formats
      format: aws-credentials-file
    ...
```

Reply:
great, thx